BlazarUI/app/scripts/components/shared/BuildingIcon.jsx

```diff
@@ -3,7 +3,13 @@ import React, {Component, PropTypes} from 'react';
 class BuildingIcon extends Component {
 
   getClassNames() {
-    return `building-icon sidebar__active-building-icon building-icon--${this.props.result} ${this.props.classNames} building-icon--${this.props.size}`;
+    let prevBuildStateModifier = ``;
+
+    if (this.props.result === "IN_PROGRESS" && this.props.prevBuildState) {
```

Comment:
i'd use the [build states](https://github.com/HubSpot/Blazar/blob/ui_v2/BlazarUI/app/scripts/constants/BuildStates.js) constants here. 

eg
`if (this.props.result === BuildStates.IN_PROGRESS && ...`